marklogic-data-hub-central/build.gradle

```diff
@@ -38,6 +39,7 @@ ext.junitJupiterVersion = '5.3.1'
 
 bootWar {
     baseName = springBootJarName
+    launchScript()
```

Comment:
Can you include a comment here indicating why this is needed - i.e. so we can run it as a service? 

I think Spring bean declarations and Gradle configuration blocks are similar - both can get a lot done in a small amount of code, but there's often a question of why the code exists. It's really slick that all we need is "launchScript()" here, but it begs the question of why we need it.